VirtoCommerce.Storefront/Domain/Customer/ExperienceMemberService.cs

```diff
@@ -92,9 +102,15 @@ public async Task DeleteContactAsync(string contactId)
 
         public async Task<Contact> GetContactByIdAsync(string contactId)
         {
+            var userId = _workContextAccessor.WorkContext?.CurrentUser?.Id;
```

Comment:
I think it is better to create one method `GetCurrentUserId` with this content and use it in all places.